benchmarks/benchmarks/signal_filtering.py

```diff
@@ -3,12 +3,29 @@
 import numpy as np
 
 try:
-    from scipy.signal import lfilter, firwin
+    from scipy.signal import lfilter, firwin, decimate
```

Comment:
Usually best to put these in a separate import statement, so that not having decimate does not prevent running eg the firwin benchmarks